test/Integration/GetWorksByPublicKey.ts

```diff
@@ -122,4 +122,12 @@ export class GetWorksByPublicKey {
       Expect(claims[i].dateCreated.toISOString()).toBe(expectedClaims[i].dateCreated.toISOString())
     }
   }
+
+  @AsyncTest()
+  @TestCase('')
```

Comment:
I like this test!

What do you think of adding other test cases that should fail, such as passing something that isn't a string maybe?

Reply:
in the cases, when we send query or params values, we only have the option to pass `string` or `empty` values, because body parser will convert to a string. For this reason, I only check empty option.

Do you have any idea?

Reply:
Makes sense. After all, all query and body params are strings really, and then we convert them to whatever.